test/e2e/toolchaincluster_test.go

```diff
@@ -36,7 +36,7 @@ func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wa
 		// given
 		name := "new-ready-" + string(await.Type)
```

Comment:
this one as well right? 
```suggestion
		name := "new-ready-" + string(otherAwait.Type)
```

Reply:
ah right! missed that! thx!